edi_oca/models/edi_exchange_record.py

```diff
@@ -39,9 +39,22 @@ class EDIExchangeRecord(models.Model):
         model_field="model",
     )
     related_name = fields.Char(compute="_compute_related_name", compute_sudo=True)
-    exchange_file = fields.Binary(attachment=True)
+    # here we store exchange file when needed (when no parent hold the file)
+    real_exchange_file = fields.Binary(attachment=True)
+    # here we give the exchange file (from parent exchange or self)
+    exchange_file = fields.Binary(
+        store=False,
+        compute="_compute_exchange_file",
+        inverse="_inverse_exchange_file",
+        attachment=True,
```

Comment:
No need of attachment as it has compute and inverse...
store false is not necessary

Reply:
This is pretty advanced orm knowledge I don't have, thanks